content/en/docs/ops/ambient/install/helm-installation/index.md

```diff
@@ -88,10 +88,10 @@ After installing all the components, you can check the Helm deployment status wi
 {{< text syntax=bash snip_id=show_components >}}
 $ helm ls -n istio-system
 NAME            NAMESPACE       REVISION    UPDATED         STATUS      CHART           APP VERSION
-istio-base      istio-system    1           ... ... ... ... deployed    base-1.0.0      1.0.0
-istio-cni       istio-system    1           ... ... ... ... deployed    cni-1.0.0       1.0.0
-istiod          istio-system    1           ... ... ... ... deployed    istiod-1.0.0    1.0.0
-ztunnel         istio-system    1           ... ... ... ... deployed    ztunnel-1.0.0   1.0.0
+istio-base      istio-system    1           ... ... ... ... deployed    base-{{< istio_full_version >}}     {{< istio_full_version >}}
```

Comment:
It would be nice if `_verify_like` handles the UPDATED fields as well so we wouldn't need `...` there either.

Reply:
Looking into that as well. From the test output I see we might only have to accept things like `2024-04-17` and `22:14:46.334153508`. The other fields seem to match.